submissions/Roka20012/pop-up-menu/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head> 
+    <meta charset="UTF-8">
+    <title>HTML_CSS_POPUP_MENU</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/ring.png">
+  </head>
+  <body>
+    <div class="container">
+      <input class="check control" type="checkbox">
+      <div class="mark"></div>
+      <input id="check-show-more" type="checkbox">
+      <div class="container-menu">
+        <div class="sprite-container"><a href="">
+            <div class="sprite"></div><span>Phone</span></a><a href="">
+            <div class="sprite"></div><span>Guard</span></a><a href="">
+            <div class="sprite"></div><span>Document</span></a><a href="">
+            <div class="sprite"></div><span>Maps</span></a><a href="">
+            <div class="sprite"></div><span>Excel</span></a><a href="">
+            <div class="sprite"></div><span>Mark</span></a><a href="">
+            <div class="sprite"></div><span>I`m off</span></a><a href="">
+            <div class="sprite"></div><span>Locked</span></a><a href="">
+            <div class="sprite"></div><span>No no..</span></a><a href="">
+            <div class="sprite"></div><span>School?</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>I`m off</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>Locked</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>No no..</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>School?</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>I`m off</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>Locked</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>No no..</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>School?</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>I`m off</span></a><a class="hide" href="">
+            <div class="sprite"></div><span>Locked</span></a><a class="hide more" href="">More from...</a></div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Add an empty line at the end of the file.

Reply:
Alredy done)